### PR TITLE
Feature/favorite recipes display

### PR DESCRIPTION
**Pull request by tfettig22**
What's this PR do?
- Update filterByName function so that it now works as intended by not being case-sensitive
- Add CSS grid to ingredients list area so that we can see quantities, name, and cost of each ingredient in a grid
Where to start?
- test filter function by loading webpage and typing into search bar, with search by name button checked

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -81,14 +81,12 @@ <h2>Foodies  Welcome</h2>
                 <div class="ingredient-title-wrapper">
                    <p class="ingredient-title">Ingredients</p>
                 </div>
-                <div class="ingredients-wrapper">
-                   <p class="ingredients-listed" id="ingredientsListed">listedIngredients</p>
-                </div>
-              </section>
-              <div class="price-list-wrapper" id="priceListWrapper">
-                <p class="price-list-container" id="priceListContainer"></p>
+                <section class="ingredient-boxes-container">
```
Comment (GOECHA) [inline dist/index.html:84]: Great use of flex grid!

File: src/scripts.js [hunk 2]
```diff
@@ -190,12 +193,18 @@ function filterRecipeByTag(tag) {
 }
 
 function filterRecipeByName(name) {
-  // let input = name.toLowerCase()
-  // let lowerCaseRecipes = recipeRepository.recipeData
-  // lowerCaseRecipes.forEach(recipe => {
-  //   recipe.name = recipe.name.toLowerCase()
-  // })
-  let filteredRecipes = recipeRepository.filterByName(name);
+  let input = name.toLowerCase()
```
Comment (GOECHA) [inline src/scripts.js:196]: Great use of .toLowerCase() to make searching consistent with data info.

**Review by GOECHA — APPROVED** [review]
Great exploration of flex grid and use of function input for consistent data entry matching.

**Review by forsethnico — APPROVED** [review]
Nice job getting the filter to work with the upper and lower case recipe names.

**Pull request merged**